### DISCUSSION TITLE
individual live logs for multiple containers in a single yaml

### DISCUSSION THREAD

**Discussion by tsentekin** [louislam/dockge#325] [Feature Request]
### ⚠️ Please verify that this feature request has NOT been suggested before.

- [X] I checked and didn't find similar feature request

### 🏷️ Feature Request Type

UI Feature

### 🔖 Feature description

Is there a way to add separate live log output for multiple containers deployed with a single compose file? current live log is a single console output and shows mixed logs for all the containers upped with the same compose file.

This is a great piece of work you've done so far, thanks.

### ✔️ Solution

It would be great if we can click on the deployed containers and they default to their own live logs when highlighted etc.

### ❓ Alternatives

Any alternatives are wellcome

**Comment by Hutch79** [comment]
Seams like a duplicate of #123 
Separation of the console is also part of this.
